mazegame/Assets/Scripts/Menu/MenuText.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.SceneManagement;
+
+public class MenuText : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler, IPointerDownHandler
+{
+    
+
+    public void OnPointerEnter(PointerEventData eventData)
+    {
+        this.transform.localScale += new Vector3(0.4f, 0.4f, 0.4f);
```

Comment:
please assign this to a private class variable. Such as `private Vector3 text_scaling = Vector3(0.4f, 0.4f, 0.4f);`